`box` command not found

After a clean installation with php 7.1, Vbox 5.1.10 & Vagrant 1.9.0

```
➜  manalize git:(master) make build
Loading composer repositories with package information
Updating dependencies
Dependency resolution completed in 0.129 seconds
Analyzed 3130 packages to resolve dependencies
Analyzed 50606 rules to resolve dependencies
Dependency resolution completed in 0.000 seconds
  - Installing elao/enum (v0.4.0)
    Downloading
    Extracting archive

  - Installing composer/semver (1.4.2)
    Downloading
    Extracting archive

  - Installing nikic/iter (v1.4.0)
    Downloading
    Extracting archive

  - Installing guzzlehttp/promises (1.3.0)
    Downloading
    Extracting archive

  - Installing psr/http-message (1.0.1)
    Downloading
    Extracting archive

  - Installing guzzlehttp/psr7 (1.3.1)
    Downloading
    Extracting archive

  - Installing guzzlehttp/guzzle (6.2.2)
    Downloading
    Extracting archive

  - Installing symfony/yaml (v3.2.0)
    Downloading
    Extracting archive

  - Installing symfony/process (v3.2.0)
    Downloading
    Extracting archive

  - Installing symfony/filesystem (v3.2.0)
    Downloading
    Extracting archive

  - Installing psr/log (1.0.2)
    Downloading
    Extracting archive

  - Installing symfony/debug (v3.2.0)
    Downloading
    Extracting archive

  - Installing symfony/polyfill-mbstring (v1.3.0)
    Downloading
    Extracting archive

  - Installing symfony/console (v3.2.0)
    Downloading
    Extracting archive

Writing lock file
Generating optimized autoload files
/bin/sh: box: command not found
make: *** [build] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -27,6 +27,6 @@ build:
 	mkdir .build
 	cp -r ./* .build
 	composer install --verbose --no-progress --no-interaction --prefer-dist --optimize-autoloader --no-dev --working-dir=.build/
-	cd .build && box build -v && cd ../
+	cd .build && ../vendor/bin/box build -v && cd ../
 	cp .build/manalize.phar .
 	rm -rf .build

--- a/composer.json
+++ b/composer.json
@@ -28,7 +28,8 @@
         "symfony/console": "~3.2@dev",
         "phpunit/phpunit": "~5.4",
         "symfony/phpunit-bridge": "~2.8|~3.0",
-        "friendsofphp/php-cs-fixer": "2.0.x@dev"
+        "friendsofphp/php-cs-fixer": "2.0.x@dev",
+        "kherge/box": "^2.7"
     },
     "autoload": {
         "psr-4": { "Manala\\Manalize\\": "src/" }

